doc/gettingstarted.md

```diff
@@ -91,319 +97,555 @@ $ ttn-lw-cli login
 
-## <a name="mqtt">Using the MQTT broker</a>
+## <a name="mqtt">Using the MQTT server</a>
 
-In order to use the MQTT broker it is necessary to register a new API key that will be used during the authentication process:
+In order to use the MQTT server you need to create a new API key to authenticate:
 
 ```bash
-$ ttn-lw-cli app api-keys create --application-id app1 --right-application-traffic-down-write --right-application-traffic-read
+$ ttn-lw-cli applications api-keys create \
+  --application-id app1 \
+  --right-application-traffic-read \
+  --right-application-traffic-down-write
 ```
 
-Note that this new API key can both receive uplinks and schedule downlinks. You can now login using an MQTT client using the username `app1` (the application name) and the newly generated API key as password.
+You can now login using an MQTT client with the application ID `app1` as user name and the newly generated API key as password.
 
-There are many MQTT clients available; a simple one is `mosquitto_pub` and `mosquitto_sub`, part of [Mosquitto](https://mosquitto.org).
+There are many MQTT clients available. Great clients are `mosquitto_pub` and `mosquitto_sub`, part of [Mosquitto](https://mosquitto.org).
+
+>Tip: when using `mosquitto_sub`, pass the `-d` flag to see the topics messages get published on. For example:
+>
+>`$ mosquitto_sub -h localhost -t '#' -u app1 -P 'NNSXS.VEEBURF3KR77ZR..' -d`
 
 ### Subscribing to messages
 
-MQTT topics provided by the built-in broker follow the format `v3/{application id}/devices/{device id}/{traffic type}`. While you could indeed subscribe for separate topics, for the purpose of this tutorial we will use the wildcard topic `#`, which provides all of the available messages of the application.
+The Application Server publishes on the following topics:
+
+- `v3/{application id}/devices/{device id}/join`
+- `v3/{application id}/devices/{device id}/up`
+- `v3/{application id}/devices/{device id}/down/queued`
+- `v3/{application id}/devices/{device id}/down/sent`
+- `v3/{application id}/devices/{device id}/down/ack`
+- `v3/{application id}/devices/{device id}/down/nack`
+- `v3/{application id}/devices/{device id}/down/failed`
 
-After subscribing to `#` from your client, when a device of the application that is currently logged in joins the network, a `join` message will be published. For example, for a device called `dev-simulator`, the message will be published on the topic `v3/app1/devices/dev-simulator/join` with the following contents:
+While you could subscribe to separate topics, for the tutorial subscribe to `#` to subscribe to all messages.
+
+With your MQTT client subscribed, when a device joins the network, a `join` message gets published. For example, for a device ID `dev1`, the message will be published on the topic `v3/app1/devices/dev1/join` with the following contents:
```

Comment:
We don't show the results of our previous CLI operations. Maybe we should consistently do that (or not do it at all).

Reply:
That is consistently a return from a manual operation; this is what you get streaming. That's useful to see, because otherwise you're looking at a blank screen.

Will add in the beginning that CLI returns the created values though